from domain.common.exceptions import InValidDomainException


class InvalidOrderException(InValidDomainException):
    pass
